packages/languages/bin/download.js

```diff
@@ -1,24 +1,21 @@
+/* eslint-disable no-console, no-process-exit, import/no-nodejs-modules */
```

Comment:
This could be moved up to `./bin/.eslintrc.js`, and I think they will apply to any script inside `./bin`

Reply:
[`f50407a` (#46709)](https://github.com/Automattic/wp-calypso/pull/46709/commits/f50407add9123d769fc390f8c6c274e9bf7d473c)